data/zilliqa/assets/FLAT

```diff
@@ -0,0 +1,17 @@
+  
```

Comment:
remove this newline

the folder should be named after the hash, and this is the wrong name. look at any other token to see how its done